Integrations/AWS-EC2/AWS-EC2.py

```diff
@@ -2636,6 +2636,117 @@ def modify_image_attribute_command(args):
         demisto.results('Image attribute sucessfully modified')
 
 
+def detach_internet_gateway_command(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+    kwargs = {}
+    if args.get('InternetGatewayId') is not None:
+        kwargs.update({'InternetGatewayId': args.get('InternetGatewayId')})
+    if args.get('VpcId') is not None:
+        kwargs.update({'VpcId': args.get('VpcId')})
+
+    response = client.detach_internet_gateway(**kwargs)
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
```

Comment:
what if it dose not equal 200? what is the handling?(this dose for the other commands also)

Reply:
Actually, I'm not sure. I remember there being a reason for it, but I'll check.

Reply:
I remember the reason. If there is anything other than a 200, the boto3 client catches the error which gets returned as an exception (which we handle)